Patches/RH2 Rimmu-Nation² - Clothing/Armors_CE.xml

```diff
@@ -420,6 +422,39 @@
 					</value>
 				</li>
 
+				<!--OpsCoreFAST SF with Night Vision-->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[
+						defName="RNApparel_OpsCoreFAST_SF"]/statBases </xpath>
+					<value>
+						<NightVisionEfficiency_Apparel>0.7</NightVisionEfficiency_Apparel>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[
+						defName="RNApparel_OpsCoreFAST_SF"]/recipeMaker/researchPrerequisite </xpath>
+					<value>
+         				<researchPrerequisite>Smithing</researchPrerequisite>
```

Comment:
This will create a duplicate node error, I believe.

Reply:
It will duplicate the original recipe, I presume? This one uses PatchOperationReplace, so I thought it would replace the original prerequisite. Should I delete it?

Reply:
I'm not sure exactly what the effect will be, but the game will report duplicate error for having two `researchPrerequisite` in the same item. As far as I know, RecipeMaker doesn't support items having multiple research requirements--you need to set up a proper RecipeDef in order to do that.

In this case, you're probably fine just leaving the `CE_AdvancedNV` one and removing the Smithing one.

Reply:
Actually, I stand corrected on this--RecipeMaker used to only support one, but it can now support multiple. You'll need to use `researchPrerequisites` instead of `researchPrerequisite`, though. It'll look something like this:
```
<researchPrerequisites>
   <li>Smithing</li>
   <li>CE_AdvancedNV</li>
</researchPrerequisites>
```